src/js/view/utils/skin.js

```diff
@@ -179,7 +179,8 @@ export function handleColorOverrides(playerId, skin) {
 
         addStyle([
             '.jw-progress',
-            '.jw-knob'
+            '.jw-knob',
+            'jw-related-autoplay-progress'
```

Comment:
This will be going away because we are reverting the auto advance work

https://github.com/jwplayer/jwplayer-plugin-related/pull/183